projects/aca-content/src/lib/components/details/details.component.scss

```diff
@@ -1,19 +1,29 @@
 app-details-manager {
-  .aca-close-details-button {
-    margin-right: 15px;
-    margin-top: 2px;
-    outline: none;
-    border-radius: 4px;
-
-    &:focus {
-      background-color: var(--theme-selected-background-color);
-      outline: 2px solid var(--theme-blue-button-color);
-      border-radius: 4px;
-    }
+  .acs-details-buttons {
+    display: flex;
 
-    &:focus-visible {
-      outline: 2px solid var(--theme-blue-button-color);
+    .aca-close-details-button {
+      margin-right: 15px;
+      margin-top: 12px;
+      outline: none;
       border-radius: 4px;
+
+      &:focus {
+        background-color: var(--theme-selected-background-color);
+        outline: 2px solid var(--theme-blue-button-color);
```

Comment:
Some focus styling are not working as expected 
<img width="787" alt="Zrzut ekranu 2023-09-29 o 12 04 21" src="https://github.com/Alfresco/alfresco-content-app/assets/113341662/1cbef2ef-3be2-45c4-b63d-c553b753eea9">
<img width="119" alt="Zrzut ekranu 2023-09-29 o 12 04 36" src="https://github.com/Alfresco/alfresco-content-app/assets/113341662/82fa7527-fd9c-47ab-a88d-fee3e1c16a5a">
<img width="133" alt="Zrzut ekranu 2023-09-29 o 12 04 51" src="https://github.com/Alfresco/alfresco-content-app/assets/113341662/23d2c0b5-012e-4c6b-b605-ae24816cc944">

Reply:
Same for close and save buttons for each section

Reply:
We have fixed the focus issue.